src/components/About.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+export default function About() {
+  return (
```

Comment:
## Hello @azy64 

**This is a great project you have going on, well-done**. 

### To Highlight 
:heavy_check_mark: Very descriptive commit messages
:heavy_check_mark: Linters well set up
:heavy_check_mark: Use of Gitflow :100: 
   
**However, there are a few sections to improve on** 
- [ ] It will be nice to limit the use of `eslint-disable` on your project. The purpose of linters check helps you to learn best practices.
- [ ] It will be nice to limit the use of `stylelint-disable` on your project. The purpose of linters check helps you to learn best practices.
- [ ] Please update your README.md file and make sure to add a live link to the project.

### Good luck with the progress.